app/src/ApiCalendar.js

```diff
@@ -0,0 +1,301 @@
+require("dotenv").config();
+
+//Credit: https://github.com/Kubessandra/react-google-calendar-api
+const CLIENT_ID = process.env.REACT_APP_GCAL_CLIENT_ID;
```

Comment:
In this one you have both `REACT_APP_GCAL` Make sure the names are consistent everywhere they're called.

Reply:
This one is correct, right? @sunnybrie